baseline/tf/lm/training/eager.py

```diff
@@ -13,7 +13,7 @@
 
 def loss_with_state(model, h, x, y):
     logits, h_out = model(x, h)
-    vsz = model.embeddings[model.tgt_key].get_vsz()
+    vsz = model.vsz
```

Comment:
Does it make sense to call this `vsz`? In other places we call it things like nc or num_tags. Should we standardize one something like `output_dim` for these models like we have on the layers?

Reply:
I dothink it would probably make unify the fields, however, The vocab size value (`vsz`) Is used pretty consistently elsewhere (in the embeddings), and I'd hesitate to unify all and risk breakage before the next major release.  We could potentially define `def output_dim()` properties on the models to prepare them for this, which could, in turn, delegate to the current names, which would make it easier to phase them in, but I do think that this is probably outside of the scope of the current PR